WIP/TankManagement/TankFactory.py

```diff
@@ -0,0 +1,40 @@
+from Tanks.AT_SPG import AT_SPG
+from Tanks.HEAVY_TANK import HEAVY_TANK
+from Tanks.LIGHT_TANK import LIGHT_TANK
+from Tanks.MEDIUM_TANK import MEDIUM_TANK
+from Tanks.SPG import SPG
+from Tanks.Tank import Tank
+from Aliases import jsonDict
+from Utils import HexToTuple
+
+class TankFactory:
+    """
+    A class that manages the creation of tanks.
+    """
+    def __init__(self) -> None:
+        """
+        Initializes the TankFactory.
+        """
+        self.__tankTypeMapping = {
```

Comment:
Usually it is not really a good idea to use technology-specific words in variable names. It's already pretty clear from the code itself that this is a mapping. You'd better use more domain-specific names. tanks_by_type for example.

Reply:
changed to __tankByType